src/main/java/uk/gov/hmcts/probate/service/ReprintService.java

```diff
@@ -27,7 +27,8 @@ public class ReprintService {
     public static final String LABEL_REISSUED_GRANT = "ReissuedGrant";
     public static final String LABEL_WILL = "Will";
     public static final String LABEL_SOT = "SOT";
-    public static final String WILL_DOC_TYPE = "Other";
+    public static final String WILL_DOC_TYPE = "will";
```

Comment:
are these actually the same values that you have set up as statics in Constants - reuse those?